xrpl/asyncio/transaction/main.py

```diff
@@ -79,6 +80,11 @@ async def safe_sign_transaction(
     if check_fee:
         await _check_fee(transaction)
     transaction_json = _prepare_transaction(transaction, wallet)
+    transaction_json = {
+        field: value
+        for field, value in transaction_json.items()
+        if not field.startswith(TX_FLAG_PREFIXES)
```

Comment:
What does this do?

Reply:
For now if you want to define a flag with bool you are defining an instance variable.
Because you would then have the flag name as key in the `transaction_json` I added that code to delete the flag keys.
Else you could not `encode_for_signing`.

**When I implemented the changes suggested by @mvadari that part will be omitted.**